README.md

```diff
@@ -162,16 +143,31 @@ webExt.cmd.run({
   });
 ```
 
-To get a list of connected adb devices, you may use the following function which takes an optional argument `adbBin` which is  a path to custom adb binary:
+#### Running the extension on Firefox for android:
 ```js
-webExt.util.adb.listADBDevices(); // returns a Promise<Array<string>>
-```
+// Path to adb binary (optional parameter, auto-detected if missing)
+const adbBin = "/path/to/adb";
+// Get an array of device ids (Array<string>)
+const deviceIds = await webExt.util.adb.listADBDevices(adbBin);
+const adbDevice = ...
+// Get an array of Firefox APKs (Array<string>)
+const firefoxAPKs = await webExt.util.adb.listADBFirefoxAPKs(
+  deviceId, adbBin
+);
+const firefoxApk = ...
 
-To get a list of installed firefox APK packages for an ADB client, you may use the following function which takes in deviceId(required) and adbBin(optional) as arguments:
-```js
-webExt.util.adb.listADBFirefoxAPKs(deviceId); // returns a Promise<Array<string>>
+webExt.cmd.run({
+  target: 'firefox-android',
+  firefoxApk,
+  adbDevice,
+  sourceDir: ...
+}).then((extensionRunner) => {...});
 ```
```

Comment:
By adding the section title `#### running the extension on Firefox for android` right before the new code snippet, everything after it will look like part of this particular subsection, but the other code snippets are not strictly related to this section.

To avoid reworking the README too much in this PR just for the small addition we care about, I would suggest to avoid the new section title and just put right before the snippet just that small intro paragraph originally suggested.

We may definitely evaluate to make this API doc section a bit easier to navigate, but let's do that in a follow up (mainly to ensure this is ready to merge to the main branch sooner, the other changes in this PR have higher chances to get conflicts with other PRs we may merge in the main time).